skema/rest/schema.py

```diff
@@ -81,43 +97,42 @@ class MiraGroundingInputs(BaseModel):
 
 class MiraGroundingOutputItem(BaseModel):
     class MiraDKGConcept(BaseModel):
-        id: str = Field(
-            description="DKG element id",
-            example="apollosv:00000233"
-        )
+        id: str = Field(description="DKG element id", example="apollosv:00000233")
         name: str = Field(
-            description="Canonical name of the concept",
-            example="infected population"
+            description="Canonical name of the concept", example="infected population"
         )
         description: Optional[str] = Field(
             description="Long winded description of the concept",
-            example="A population of only infected members of one species."
+            example="A population of only infected members of one species.",
         )
         synonyms: List[str] = Field(
             description="Any alternative name to the cannonical one for the concept",
-            example=["Ill individuals", "The sick and ailing"]
+            example=[["Ill individuals", "The sick and ailing"]],
         )
         embedding: List[float] = Field(
             description="Word embedding of the underlying model for the concept"
         )
 
+        def __hash__(self):
+            return hash(tuple([self.id, tuple(self.synonyms), tuple(self.embedding)]))
+
     score: float = Field(
         description="Cosine similarity of the embedding representation of the input with that of the DKG element",
-        example=0.7896
+        example=0.7896,
     )
     groundingConcept: MiraDKGConcept = Field(
         description="DKG concept associated to the query",
-        example=MiraDKGConcept(
-            id="apollosv:00000233",
-            name="infected population",
-            description="A population of only infected members of one species.",
-            synonyms=[],
-            embedding=[
-                0.01590670458972454,
-                0.03795482963323593,
-                -0.08787763118743896,
-            ]
-        )
+        # example=MiraDKGConcept(
```

Comment:
@enoriega , if we uncomment this, are the docs negatively affected?

Reply:
It shouldn't. If I comment it I can see it in swagger and it doesn't break the docs

<img width="747" alt="image" src="https://github.com/ml4ai/skema/assets/4502559/fff5c30f-477c-40e9-a63f-c6834362021c">